.gemspec

```diff
@@ -38,6 +38,6 @@ Gem::Specification.new do |s|
   s.rubyforge_project = "puppet"
   s.summary = "Puppet, an automated configuration management tool"
   s.specification_version = 3
-  s.add_runtime_dependency(%q<facter>, [">= 1.7", "< 3"])
+  s.add_runtime_dependency(%q<facter>, [">= 1.7", "< 4"])
```

Comment:
One thought I had was what if someone installs puppet as a gem, but (ruby) facter 2.7.5 will still match the constraints and we should be fine, so :+1: